common/djangoapps/student/views.py

```diff
@@ -1284,9 +1284,12 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
     user_found_by_email_lookup = user
     if user_found_by_email_lookup and LoginFailures.is_feature_enabled():
         if LoginFailures.is_user_locked_out(user_found_by_email_lookup):
+            overest_lockout_mins=str(int(settings.MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_SECS*.0222+1)) #cl .0222 is 33% more with the conversion into minutes.  1.33/60=0.02216
```

Comment:
Since this value never changes, we should avoid recalculating the math.
Let's move this up to the top level as a local constant.

```python
OVERESTIMATE_LOCKOUT_MINUTES = int(settings.MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_SECS*.0222+1)
```